src/signals/incident/components/form/ContainerSelect/Selector/Selector.tsx

```diff
@@ -89,6 +96,28 @@ const EmptySelectionWrapper = styled.div`
   margin: ${themeSpacing(4)} 0;
 `;
 
+const ButtonBarStyle = styled.div<{ layerVisible: boolean }>`
+  @media screen and ${breakpoint('max-width', 'tabletM')} {
```

Comment:
This should be 

`  @media screen and (${breakpoint('min-width', 'tabletM')}) {`

Or it won't be triggerred on Mac (also, I get a tslint error for it: `( expected ts-styled-plugin(9999)`)

Reply:
Good catch! Added brackets! I get that the ts error as well. Not found yet a way to suppress it. It looks that with brackets is gone :-)